spec/models/manageiq/providers/nuage/network_manager/vcr_specs/refresher_spec.rb

```diff
@@ -201,9 +201,10 @@ def assert_cloud_subnets
       :network_router_id              => NetworkRouter.find_by(:ems_ref => router_ref).id,
       :parent_cloud_subnet_id         => nil,
       :extra_attributes               => {
-        "domain_id" => "75ad8ee8-726c-4950-94bc-6a5aab64631d",
-        "zone_name" => "Zone 1",
-        "zone_id"   => "6256954b-9dd6-43ed-94ff-9daa683ab8b0"
+        "domain_id"   => "75ad8ee8-726c-4950-94bc-6a5aab64631d",
+        "zone_name"   => "Zone 1",
+        "zone_id"     => "6256954b-9dd6-43ed-94ff-9daa683ab8b0",
+        "template_id" => nil
```

Comment:
ah, not sure, it will probably be buried somewhere in the nuage gem. Isn't it easier to just rerecord the VCR?

Reply:
As far as I can tell it's not Nuage who's encoding binary, but VCR. Because there is no gem for nuage: we're developing each REST API call manually using `RestClient::Request` and there is no binary encoding there...

I'll play a little more with decoding tomorrow, I'd prefer modify this VCR or else we might have problems with other tests because of all the relations and stuff...

Reply:
ah, is it, I thought VCR just records whatever is in the HTTP request/response

e.g. for AWS I see https://github.com/Ladas/manageiq-providers-amazon/blob/ea2a55b7aa2097ac728104c32e27af56ffba8602/spec/vcr_cassettes/manageiq/providers/amazon/cloud_manager/refresher_targeted/vpc_vm_with_public_ip_and_template.yml#L44

Reply:
Turns out Nuage sends gzipped json string that can be decompressed e.g. here http://www.txtwizard.net/compression. What I did is I decompressed it manually and removed 'gzip' Content-Encoding response header and everything seems to work. Plus I'm now able to tweak `templateID` in response 😎 

Note this is just to decode existing gzipped content, VCR now stores uncompressed string due to https://github.com/ManageIQ/manageiq-providers-nuage/blob/master/spec/spec_helper.rb#L13